app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    content = command.split()
+    command, old_file, new_file = content[0], content[1], content[2]
```

Comment:
If splited command length less that 3 elements your program will not work
You must validate this